### PR TITLE
test: update cypress dependencies

### PR DESCRIPTION
**Pull request by DlgSHi**
### Proposed behaviour

- update dependencies
- update `docker-compose.yml` file

### Current behaviour

- an old version of cypress

### Checklist

- [x] Commits follow our style guide
- [ ] Related issues linked in commit messages if required
- [ ] Screenshots are included in the PR if useful
- [ ] All themes are supported if required
- [ ] Unit tests added or updated if required
- [x] Cypress automation tests added or updated if required
- [ ] Storybook added or updated if required
- [ ] Translations added or updated (including creating or amending translation keys table in storybook) if required
- [ ] Typescript `d.ts` file added or updated if required
- [ ] Related docs have been updated if required

#### QA

- [ ] Tested in CodeSandbox/storybook
- [ ] Add new Cypress test coverage if required
- [ ] Carbon implementation matches Design System/designs
- [ ] UI Tests GitHub check reviewed if required

### Testing instructions
- [x] All tests should pass

### CODE REVIEW

File: src/components/advanced-color-picker/advanced-color-picker.test.js [hunk 1]
```diff
@@ -84,17 +85,13 @@ context("Testing AdvancedColorPicker component", () => {
...
-      simpleColorPicker(0).should("not.exist");
-    });
-
-    it("should close AdvancedColorPicker using Enter on selected color", () => {
-      simpleColorPicker(7).trigger("keydown", keyCode("Enter"));
-
-      simpleColorPicker(0).should("not.exist");
-    });
+    it.each(["Space", "Enter"])(
+      "should close AdvancedColorPicker using Space on selected color",
```
Comment (stephenogorman) [inline src/components/advanced-color-picker/advanced-color-picker.test.js:89]: "should close AdvancedColorPicker using %s on selected color"

File: src/components/flat-table/flat-table.test.js [hunk 2]
```diff
@@ -3199,12 +3217,17 @@ context("Tests for Flat Table component", () => {
         .should("have.css", "background-color", "rgb(25, 71, 94)");
     });
 
-    it("should render Flat Table Header and Cells with custom vertical header and body borders", () => {
+    it.only("should render Flat Table Header and Cells with custom vertical header and body borders", () => {
```
Comment (stephenogorman) [inline src/components/flat-table/flat-table.test.js:3220]: it("should render Flat Table Header and Cells with custom vertical header and body borders", () => {

File: src/components/flat-table/flat-table.test.js [hunk 3]
```diff
@@ -3199,12 +3217,17 @@ context("Tests for Flat Table component", () => {
...
       CypressMountWithProviders(<FlatTableVerticalBordersComponent />);
 
       flatTableHeader()
         .find("th")
         .eq(0)
+        // .then(($el) => {
+        //   expect(parseInt($el.css("border-right-width"))).to.equals(
+        //     parseInt(borderSizeSmall)
+        //   );
+        // });
```
Comment (stephenogorman) [inline src/components/flat-table/flat-table.test.js:3230]: should commented lines be here?

**Review by stephenogorman — APPROVED** [review]

**Review by grabkowski — CHANGES_REQUESTED** [review]
I think that the commit message and PR title should be:
`test: update cypress dependencies`

**Review by grabkowski — APPROVED** [review]

**Review by stephenogorman — APPROVED** [review]

**Review by ZhuoyuJin — APPROVED** [review]

**Pull request merged**